README.md

```diff
@@ -45,10 +45,14 @@ The output of the second command should provide you with the base URL for callin
 
 > Note: It can take up to five minutes for the API configuration to be fully setup and ready for use
 
-## Run Tests
+## Run Unit Tests
 
 `npm test`
 
+## Functional Testing
+
+To functionally test the API, developers can import the Template Registry collection [template-registry-collection.json](https://github.com/adobe/adp-template-registry-api/blob/main/template-registry-api.json) into [Insomnia](https://insomnia.rest/) (or any API tooling forked from Insomnia).
```

Comment:
Looks like we are pointing to template-registry-api.json instead of template-registry-collection.json in the hyperlink